src/types/users.ts

```diff
@@ -17,6 +17,28 @@ export type UserTableData = {
 	isVerified: boolean
 	isActive: boolean
 	isDeleted: boolean
-}
+} & Range & OrderDir
 
 export type UserDetails = Omit<UserTableData, "password">
+
+
+export type UserUpdatePayload = {
+	userId: string
+} & Partial<{
+	name: string
+	email: string
+	roleId: number
+	mobile: string
+	password: string
+	dob: string
+	address: string
+	city: string
+	state: string
+	country: string
```

Comment:
We are not updating isVerified or some fields lke that.
Please use the type in auth file.